### PR TITLE
Lili2

### PR DESCRIPTION
**Pull request by Liliia16**
added new description

### CODE REVIEW

File: homeworklili.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
+1. B
+2. B
+3. A
+4. B
+5. B
+6. A
+7. A
+8. B
+9. A
```
Comment (nromanen) [inline homeworklili.txt]: indent

**Review by alexlotozka — COMMENTED** [review]
somethigs is wrong

**Review by IgorBloshko — COMMENTED** [review]
good work!

File: homeworklili.txt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+10. A
+11. A
```
Comment (Alexwork4444) [inline homeworklili.txt]: Test comment

File: homeworklili.txt [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+12. A
+13. B
+14. A
+15. B
+16. A
```
Comment (alexlotozka) [inline homeworklili.txt]: my comment

File: homeworklili.txt [hunk 4]
```diff
@@ -0,0 +1,35 @@
+1. B
+2. B
+3. A
```
Comment (Alexwork4444) [inline homeworklili.txt]: test

**Pull request merged**